ndk/Cargo.toml

```diff
@@ -17,6 +17,7 @@ all = ["audio", "bitmap","media", "api-level-30"]
 audio = ["ffi/audio", "api-level-26"]
 bitmap = ["ffi/bitmap"]
 media = ["ffi/media"]
+midi = ["ffi/midi", "api-level-29"]
```

Comment:
Don't these exist since API level 23?

Reply:
Looks like you're right, I may have ended up accidentally scrolling the Java docs and seeing all the `Available since API 23` notes.

The NDK docs don't mention a bound at all, except for [`AMidiDevice_getDefaultProtocol` which is set at 33](https://developer.android.com/ndk/reference/group/midi#amididevice_getdefaultprotocol), so let's keep these all at `29` :+1: